packages/office-ui-fabric-react/src/demo/pages/ButtonPage/ButtonPage.tsx

```diff
@@ -55,9 +53,6 @@ export class ButtonPage extends React.Component<IComponentDemoPageProps, IButton
             <ExampleCard title='Primary Button' code={ ButtonPrimaryExampleCode }>
```

Comment:
Also on this page, the "Button with Aria Description for Screen Reader" example still uses `<Button>` when it should be `<PrimaryButton>`.